MantidPlot/src/main.cpp

```diff
@@ -207,7 +207,7 @@ int main(int argc, char **argv) {
   try {
     // Splash
     QPixmap pixmap;
-    if (!pixmap.load(":/MantidSplashScreen.png"))
+    if (!pixmap.load(":/MantidPlotSplashScreen-Deprecated.png"))
```

Comment:
End of life does not specify a date easily to hand. Would it be better to say something like `final stable release` or provide a date?

Reply:
We told by the PMB not to specify a date, as they were concerned that is would delay scientists moving over until the date is imminent

However we need to make it prominent that action to move over is important and final stable release does not really suggest that.  The PMB want it to be very clear on the splash screen that this  is the end.